simulation/tools/CMakeLists.txt

```diff
@@ -1,31 +1,39 @@
 set(SUBSYS_NAME tools)
 
-find_package(GLEW)
+if(APPLE)
+  # glew in homebrew installs glew-config.cmake
+  find_package(glew)
+else()
+  find_package(GLEW)
+endif()
 find_package(GLUT)
 
 if(NOT (GLEW_FOUND AND GLUT_FOUND))
   return()
 endif()
 
-include_directories(SYSTEM ${VTK_INCLUDE_DIRS})
+include_directories(${VTK_INCLUDE_DIRS})
+
+set(_glut_target $<IF:${APPLE},${GLUT_glut_LIBRARY},GLUT::GLUT>)
```

Comment:
The library path is not properly set in `GLUT::GLUT` so I explicitly go for `${GLUT_glut_LIBRARY}`

Reply:
Why remove SYSTEM from VTK_INCLUDE_DIRS?

Reply:
Forgot to comment of it: it's a precautionary measure stemming from this comment
https://github.com/Homebrew/homebrew-core/pull/59146#issuecomment-670649822

Reply:
Is this indeed VTK_INCLUDE_DIRS? Should that not be `<brew_prefix>/usr/include/vtk/` instead of `<MacOS>/usr/include/`?

Reply:
I'd make it a separate PR so it could be undone easily (just in case)

Reply:
I'm not understanding the point you're trying to make. It sounds like it's related to the comment I posted though.

VTK_INCLUDE_DIRS includes vtk's own include directories and everything it depends on. There's a chance (I haven't confirmed) that  VTK_INCLUDE_DIRS also include general system header directories in that list. 

By removing the SYSTEM flag what I'm doing is saying the include dirs should no longer be treated as system header dirs. Meaning that all warning will be enabled. According to the url I linked, it seams it also affects the search path order for headers. 

I will split it from this PR nevertheless.

Reply:
> There's a chance (I haven't confirmed) that VTK_INCLUDE_DIRS also include general system header directories in that list

I see. In that case, `message(STATUS ${VTK_INCLUDE_DIRS})` on your local system would help confirm your suspicions